templates/galaxy/config/tool_conf.xml.j2

```diff
@@ -275,6 +275,12 @@
   </section>	
   <section id="hca_sc_scmap_tools" name="SCMap">
   </section>	
+  <section id="hca_sc_scpred_tools" name="SCPred">
```

Comment:
you can be more descriptive in the naming ...

Reply:
Will ask the person involved in these tools, thanks for taking a look.

Reply:
```suggestion
  <section id="hca_sc_scpred_tools" name="ScPred - scRNAseq cell type classification">
```